features/share/api/src/main/kotlin/io/element/android/features/share/api/ShareService.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.share.api
+
+import kotlinx.coroutines.CoroutineScope
+
+interface ShareService {
```

Comment:
The `*Service` suffix makes me think it should be implemented by extending an Android Service. Maybe `Helper`? Or leave it as is, it's not a big issue.

Reply:
We already have a lot of `Service` classes, the rule we had on EA for classes extending `android.app.Service` is to name them with `AndroidService` suffix, you can see all the children of [VectorAndroidService](https://github.com/element-hq/element-android/blob/develop/vector/src/main/java/im/vector/app/core/services/VectorAndroidService.kt#L28) for instance.